### ISSUE TITLE
最新版的 webstrom 会经常报错

### ISSUE THREAD

**Issue by tlerbao** [boneVidy/json2ts#52]
![CleanShot 2024-09-23 at 09 08 34@2x](https://github.com/user-attachments/assets/2d6227c0-f5f0-49bb-bdc3-6ebcbc065526)

```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.json2ts.actions.Json2tsFile' must override `getActionUpdateThread()` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: com.jsonts.kotlin]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:201)

```

**Comment by tlerbao** [comment]
1.0.52 依然报错

com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.json2ts.actions.Json2ts' must override `getActionUpdateThread()` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: com.jsonts.kotlin]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:201)

**Comment by boneVidy** [comment]
＠[tlerbao](https://github.com/tlerbao)
麻烦告知下IDE 版本

**Comment by tlerbao** [comment]
![CleanShot 2024-10-17 at 09 59 57@2x](https://github.com/user-attachments/assets/1d307074-a39e-41c5-9069-64b31d75759a)

**Comment by lupillow** [comment]
Error is still present

![image](https://github.com/user-attachments/assets/8a5c5a9d-f9ec-4096-9b64-21b96ce37fc8)

WebStorm 2024.2.3
Build #WS-242.23339.15, built on September 25, 2024
Licensed to Jose Estrada
Subscription is active until June 18, 2025.
For educational use only.
Runtime version: 21.0.4+13-b509.17 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.windows.WToolkit
Windows 11.0
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 2022M
Cores: 12
Registry:
  ide.balloon.shadow.size=0
  debugger.watches.in.variables=false
  debugger.new.tool.window.layout=true
  ide.experimental.ui=true
  i18n.locale=
  terminal.new.ui=true
Non-Bundled Plugins:
  com.arcticicestudio.nord.jetbrains (0.13.0)
  com.jsonts.kotlin (1.0.52)
  com.markskelton.one-dark-theme (5.12.0)
  mobi.hsz.idea.gitignore (4.5.4)
  com.mallowigi (97.0.0)
  zielu.gittoolbox (600.0.2+242)

**Comment by AtLastDopamine** [comment]
我也是这个错误，每次打开项目 80% 的概率会报错，但不影响使用。
我也是 Mac 英特尔的 CPU。

最后感谢作者，你的这个插件真的很好用。

**Comment by boneVidy** [comment]
The issue has been fixed in version 1.0.53．
Closes #54
